### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.11 es 2 (18 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#329] [Esercizi M-Z]
![es.2](https://github.com/user-attachments/assets/732bc9c4-7066-44b0-b2bb-c82b749c94da)

**Comment by alem1105** [comment]
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/574608a1-a6a9-48ac-9c53-e59ea0246ed0">
